jumpscale/servers/gedis/baseactor.py

```diff
@@ -1,18 +1,23 @@
 import inspect
 import sys
 from functools import wraps
+from jumpscale.loader import j
 
 
 def actor_method(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         # verify args and kwargs types
         signature = inspect.signature(func)
-        bound = signature.bind(*args, **kwargs)
+        try:
+            bound = signature.bind(*args, **kwargs)
+        except TypeError as e:
+            raise j.exceptions.Value(str(e))
```

Comment:
maybe we can show the full traceback or a better exception msg instead of just the exception to make debugging easy?

Reply:
the bind method checks the passed parameters is enough to call the function or not, so the error would be missing parameter x or unexpected parameter y, so the stacktrace is not needed